website/docs/r/postgresql_virtual_network_rule.html.markdown

```diff
@@ -84,7 +85,7 @@ The following arguments are supported:
 
 * `subnet_id` - (Required) The ID of the subnet that the PostgreSQL server will be connected to.
 
-~> **NOTE:** Due to [a bug in the Azure API](https://github.com/Azure/azure-rest-api-specs/issues/3719) this resource currently doesn't expose the `ignore_missing_vnet_service_endpoint` field and defaults this to `false`. Terraform will check during the provisioning of the Virtual Network Rule that the Subnet contains the Service Rule to verify that the Virtual Network Rule can be created.
+* `ignore_missing_vnet_service_endpoint` - (Optional) Create the virtual network rule before the subnet has the virtual network service endpoint enabled. The default value is false.
```

Comment:
```
Create the virtual network rule before the subnet has the virtual network service endpoint enabled. The default value is false.
```

I think we could phrase this better to match the syntax of the other fields:

```
Should the Virtual Network Rule be created before the Subnet has the Virtual Network Service Endpoint enabled? Defaults to `false`
```

Reply:
How about I copy it from the API docs directly?

```
Create firewall rule before the virtual network has vnet service endpoint enabled.
```

Reply:
Sure - but we should still match the language used across the other resources, which prefixes things with `Should [xxx] be applied/enabled` etc?